deps/uv/include/uv-version.h

```diff
@@ -31,9 +31,9 @@
  */
 
 #define UV_VERSION_MAJOR 1
-#define UV_VERSION_MINOR 4
-#define UV_VERSION_PATCH 2
-#define UV_VERSION_IS_RELEASE 0
-#define UV_VERSION_SUFFIX "node1"
```

Comment:
Just so that we don't forget about that, if we need to float additional patches, we'll need to put back this version suffix.